packages/api-reference/src/hooks/useSidebar.ts

```diff
@@ -371,6 +374,11 @@ export const scrollToOperation = (operationId: string) => {
 export function useSidebar(options?: ParsedSpecOption & SorterOption) {
   Object.assign(optionsRef, options)
 
+  // Hack navState
```

Comment:
can you add a little bit more explanation for future us? why is that a hack, what do we need to remove it?

Reply:
oh i did above where navState is defined

Reply:
https://github.com/scalar/scalar/pull/4462/files#diff-177327574212cd0b7483ff6ea4da60e444da454855f260a9a5a2768b7a83a297R30